packages/google_maps_flutter/google_maps_flutter/ios/Classes/GoogleMapController_Test.h

```diff
@@ -0,0 +1,27 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <Flutter/Flutter.h>
+#import <GoogleMaps/GoogleMaps.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface FLTGoogleMapController (Test)
+
+/**
+ * Initialize a map controller with a concrete map view.
+ *
+ * @param mapView A map view that will be displayed by the controller
+ * @param viewId A unique identifier for the controller (unused).
```

Comment:
What does the (unused) mean? We did store it in an ivar: `_viewId = viewId;`
Does it mean the ivar itself is unused in the current implementation? I think this makes the reader confused. We should just remove the `(unused)` part. If the ivar `_viewId` is currently not accessed in the implementation, we can just remove the ivar itself. (can be in a different PR)

Reply:
Yeah, the ivar isn't actually being used in any way. I removed the comment.